packages/Babylonian-Printbugger.package/BPLabelMorph.class/instance/downsized.st

```diff
@@ -0,0 +1,6 @@
+actions
+downsized
+
+	| stringMorph |
+	stringMorph := (self submorphNamed: #lineName).
+	stringMorph fontName: (TextStyle defaultFont name asSymbol) pointSize: 8.5.
```

Comment:
does the submorph really need to be saved as a temp variable here? can't you call fontName:pointSize: on (...) directly?